components/var_dumper/advanced.rst

```diff
@@ -181,6 +186,21 @@ method. They also typically implement the
 them from re-implementing the logic required to walk through a
 :class:`Symfony\\Component\\VarDumper\\Cloner\\Data` object's internal structure.
 
+The HTMLDumper limits string length and nesting depth of the output. 
```

Comment:
We should enclose HTMLDumper with double backticks and change it to `HtmlDumper` to reflect the actual class name.